index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
```

Comment:
colours of the tetris are given here

Reply:
the colours used should be of more contrast so that they can be easily viewed during the game